harvard-de-montfort-university.csl

```diff
@@ -134,88 +127,158 @@
+              <text macro="author-short"/>
+              <text macro="year-date"/>
+            </group>
+            <group>
+              <label variable="locator" form="short"/>
+              <text variable="locator"/>
+            </group>
+          </group>
+        </else>
+      </choose>
     </layout>
   </citation>
   <bibliography hanging-indent="true" et-al-min="4" et-al-use-first="1">
     <sort>
       <key macro="author"/>
       <key variable="title"/>
     </sort>
-    <layout>
-      <text macro="author" font-variant="small-caps" suffix=","/>
-      <date variable="issued" prefix=" (" suffix=")">
-        <date-part name="year"/>
-      </date>
+    <layout suffix=".">
       <choose>
-        <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
-          <group prefix=" " delimiter=" " suffix=",">
-            <text macro="title"/>
-            <text macro="edition"/>
-            <text macro="editor"/>
+        <if type="motion_picture" match="any">
+          <group delimiter=". ">
+            <text variable="title" font-style="italic"/>
+            <group delimiter=" ">
+              <date date-parts="year" form="text" variable="issued" prefix="(" suffix=")"/>
+              <text variable="note" suffix="."/>
+              <text variable="genre" text-case="capitalize-first" prefix="[" suffix="]"/>
+              <text term="director" text-case="capitalize-first"/>
+              <text macro="author"/>
+            </group>
+            <text macro="publisher"/>
           </group>
-          <text prefix=" " suffix="." macro="publisher"/>
         </if>
-        <else-if type="chapter paper-conference" match="any">
-          <text macro="title" prefix=" " suffix="."/>
-          <group prefix=" " delimiter=" ">
-            <text term="in" text-case="capitalize-first" suffix=":"/>
-            <text macro="editor"/>
-            <text variable="container-title" font-style="italic" suffix="."/>
-            <text variable="collection-title" suffix="."/>
-            <text variable="event" suffix="."/>
-            <group suffix="." delimiter=", ">
-              <text macro="publisher" prefix=" "/>
-              <text macro="pages"/>
+        <else-if type="broadcast" match="any">
+          <group delimiter=". ">
+            <text variable="title" font-style="italic"/>
+            <group delimiter=" ">
+              <date date-parts="year" form="text" variable="issued" prefix="(" suffix=")"/>
+              <group delimiter=", ">
+                <text variable="number" prefix="Episode "/>
+                <text variable="container-title" suffix="."/>
+              </group>
+              <text variable="medium" text-case="capitalize-first" prefix="[" suffix="]"/>
+              <text variable="publisher"/>
+            </group>
+            <group delimiter=", ">
+              <date form="text" date-parts="month-day" variable="issued">
```

Comment:
this is why the validation fails -- you can either do the date-parts here or below, but not both.